### PR TITLE
Add Kubernetes Service Account Binding is Default query for Terraform #626

### PR DESCRIPTION
**Pull request by pedro-mimoso**
The default namespace should not be used #626

### CODE REVIEW

File: assets/queries/k8s/default_service_account_binding/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Default_Service_Account_Binding",
+  "queryName": "Kubernetes Service Account Binding is Default",
```
Comment (jorge-cruz) [inline assets/queries/k8s/default_service_account_binding/metadata.json:3]: I think "Kubernetes" is not necessary. A better name could be "Role Binding To Default Service Account"

File: assets/queries/k8s/default_service_account_binding/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "severity": "HIGH",
+  "category": null,
+  "descriptionText": "The default namespace should not be used",
```
Comment (jorge-cruz) [inline assets/queries/k8s/default_service_account_binding/metadata.json:6]: Change the description to "The default service account should not be used"

File: assets/queries/k8s/default_service_account_binding/query.rego [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    subjects[c].kind == "ServiceAccount"
+    subjects[c].name == "default"
+
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	   sprintf("subjects.name=%s", [subjects[c].name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("subjects.name=%s is not default", [subjects[c].name]),
+                "keyActualValue": 	sprintf("subjects.name=%s is default", [subjects[c].name])
```
Comment (jorge-cruz) [inline assets/queries/k8s/default_service_account_binding/query.rego:15]: Change the keys to:
```
"keyExpectedValue": "subjects.kind=ServiceAccount.name is not default",
"keyActualValue": "subjects.kind=ServiceAccount.name is default"
```

**Review by jorge-cruz — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/default_service_account_binding/metadata.json (hunk 2 above)
Comment (jorge-cruz) [reply assets/queries/k8s/default_service_account_binding/metadata.json:6]: Or maybe "No role nor cluster role should bind to a default service account"

**Review by jorge-cruz — CHANGES_REQUESTED** [review]

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**